app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -1005,10 +1035,10 @@ const TopicTrackingState = EmberObject.extend({
 });
 
 export function startTracking(tracking) {
-  const data = PreloadStore.get("topicTrackingStates");
-  tracking.loadStates(data);
-  tracking.establishChannels();
+  tracking.loadStates(PreloadStore.get("topicTrackingStates"));
+  tracking.establishChannels(PreloadStore.get("topicTrackingStateMeta"));
   PreloadStore.remove("topicTrackingStates");
+  PreloadStore.remove("topicTrackingStateMeta");
```

Comment:
For both of these we can just use `PreloadStore.getAndRemove`

Reply:
O nice :+1: I didn't know about this.